cellxgene_resources/curation_visium.ipynb

```diff
@@ -44,11 +62,15 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "folder = 'A2'\n",
-    "\n",
-    "#REQUIRED to include background spots, so specify raw_feature_bc_matrix.h5\n",
-    "adata = sq.read.visium(folder, counts_file='raw_feature_bc_matrix.h5')\n",
-    "adata.var.set_index('gene_ids', inplace=True)"
+    "sr_outs = '6332STDY10289523/outs'"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Create the tissue_positions_list.csv from tissue_positions.csv, if needed\\\n",
+    "Space Ranger v2.0 onwards includes tissue_positions.csv, which includes a headers, but squidpy consumes tissue_positions_list.csv "
```

Comment:
super small typo: "a headers"

Reply:
Good catch. Included the fix in the latest commit